wp-content/themes/nuvanx-medical/inc/nvx-aesthetic-treatment-catalog.php

```
@@ -0,0 +1,760 @@
+			$entry['slug'] = $slug;
+
+			$catalog[ $key ] =
+				$entry;
+
+			$seen_slugs[
+				$slug
+			] = $key;
+		}
+
+		return $catalog;
+	}
```

PHPMD (2.15.0)

[warning] The function nvx_aesthetic_treatment_catalog() has a Cyclomatic Complexity of 27. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)